BigData/BigDataTutorial.md

```diff
@@ -0,0 +1,21 @@
+ ### Big Data Tutorial
+ [Big Data University](http://bigdatauniversity.com/moodle/index.php?)
+ [Yahoo Hadoop Tutorial](https://developer.yahoo.com/hadoop/tutorial/module4.html)
+ [Blog](http://www.thecloudavenue.com/p/hadoopresources.html)
+ [Big Data Resource](http://www.bigdata-careers.com/?page_id=99)
+ [Open Data Sites](https://docs.google.com/document/pub?id=1CNBmPiuvcU8gKTMvTQStIbTZcO_CTLMvPxxBrs0hHCg)
+ [Official Apache Hadoop Sites](https://hadoop.apache.org)
+ [edureka Courses](https://www.youtube.com/user/edurekaIN/playlists?sort=dd&shelf_id=6&view=50)
+ [Online Competition For Data Science](https://www.kaggle.com/)
+ [Mapr Tutotial](http://learn.mapr.com/)
+ [HDFS Architechture](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html)
+ [Big Data Projects](https://www.acquia.com/examples-big-data-projects)
+ [Mapr Hadoop White Papers](https://www.mapr.com/resources/white-papers)
+ [Udacity Courses](https://www.udacity.com/courses/data-science)
+ [Coursera Big Data Specialization](https://www.coursera.org/specializations/big-data)
+ [Cloudera Hadoop Certification](http://www.cloudera.com/training/certification.html)
+ [Mapr Hadoop Certification](https://www.mapr.com/services/mapr-academy/certified-hadoop-developer)
+ [Hortonworks Certification](http://hortonworks.com/training/certification/)
+ <br/>
+
+ These lists is small but more will be added soon.
```

Comment:
@PSanni yes I agree! Please don't write such sentences. At this stage this repo is having small lists in each directories but all the lists are going to be big soon ;)